lucille-core/src/main/java/com/kmwllc/lucille/stage/EmitNestedChildren.java

```diff
@@ -15,16 +16,20 @@
  *
  * Config Parameters:
  * - drop_parent (Boolean, Optional): if set to true, will mark parent document as dropped. Defaults to false
- *
+ * - fields_to_copy (List<String>, Optional): list of fields to copy from parent to children
  */
 public class EmitNestedChildren extends Stage {
 
   private final boolean dropParent;
+  private final List<String> fieldsToCopy;
+  private final UpdateMode updateMode;
   private static final Logger log = LoggerFactory.getLogger(EmitNestedChildren.class);
 
   public EmitNestedChildren(Config config) {
-    super(config, new StageSpec().withOptionalProperties("drop_parent"));
+    super(config, new StageSpec().withOptionalProperties("drop_parent", "fields_to_copy", "update_mode"));
     this.dropParent = config.hasPath("drop_parent") ? config.getBoolean("drop_parent") : false;
+    this.fieldsToCopy = config.hasPath("fields_to_copy") ? config.getStringList("fields_to_copy") : List.of();
```

Comment:
this should be a map.. not a list.. so you can copy a field from the parent to a different field on the child.